src/DependencyInjection/Configuration.php

```diff
@@ -396,6 +398,88 @@ private function getFlexibleSslNode(): ArrayNodeDefinition
         return $node;
     }
 
+    private function addCrossOriginIsolationNodes(): ArrayNodeDefinition
+    {
+        $node = new ArrayNodeDefinition('cross_origin_isolation');
+        $node
+            ->canBeEnabled()
+            ->fixXmlConfig('path')
+            ->children()
+                ->arrayNode('paths')
+                    ->normalizeKeys(false)
+                    ->useAttributeAsKey('pattern')
+                    ->arrayPrototype()
+                        ->children()
+                            ->arrayNode('coep')
```

Comment:
You need to call `->addDefaultsIfNotSet()` on those array nodes (and either mark `value` as required or as having a default) to ensure that the finalized config actually define the 3 keys (which are expected by the type used in the listener).